cocos/2d/CCLayer.cpp

```diff
@@ -620,13 +620,13 @@ void LayerColor::onDraw(const Mat4& transform, uint32_t /*flags*/)
     getGLProgram()->use();
     getGLProgram()->setUniformsForBuiltins(transform);
     
-    glEnableVertexAttribArray(GLProgram::VERTEX_ATTRIB_POSITION);
-    glEnableVertexAttribArray(GLProgram::VERTEX_ATTRIB_COLOR);
-    
     //
     // Attributes
     //
     glBindBuffer(GL_ARRAY_BUFFER, 0);
+    glEnableVertexAttribArray(GLProgram::VERTEX_ATTRIB_POSITION);
+    glDisableVertexAttribArray(GLProgram::VERTEX_ATTRIB_TEX_COORD);
```

Comment:
Should disable VERTEX_ATTRIB_TEX_COORD since it is not used in the program, and it may be enabled in other places.

Reply:
It is used in vertex shader, but have no effect. If this channel is enabled, then will cause crash.